apps/zotonic_mod_authentication/priv/templates/_logon_login_extra.tpl

```diff
@@ -1,17 +1,12 @@
 {#
 Add other login options
 #}
-<ul class="z-logon-extra">
-    {% all include "_logon_extra.tpl" %}
-    <li class="text-muted z-logon-extra-separator"><span>{_ or _}</span></li>
-</ul>
-
-{# Highlight the most recently used authentication method (if any) #}
-{#
-{% if m.persistent.auth_method %}
-    {% javascript %}
-        $('.z-logon-extra li').css('opacity', '0.6');
-        $('#logon_{{ m.persistent.auth_method }}').css('opacity', '1');
-    {% endjavascript %}
-{% endif %}
-#}
+{% if not q.options.is_username_checked %}
+    <ul class="z-logon-extra">
+        {% all include "_logon_extra.tpl" %}
+        <li class="text-muted z-logon-extra-separator"><span>{_ or _}</span></li>
+    </ul>
+{% else %}
+    <ul class="z-logon-extra">
+    </ul>
+{% endif %}
```

Comment:
It looks like that need add new line in the end of file
```suggestion
{% endif %}
```suggestion
{% endif %}
```
```